lifecycle/lifecycle/config/config.py

```diff
@@ -68,6 +68,11 @@ class Config(BaseModel):
     # Retention in seconds of Async job call models
     async_job_call_lifetime: int = 4 * 3600
 
+    # Maximum number of seconds to wait until the job is alive in the cluster (/live endpoint responds)
+    timeout_until_job_alive: int = 15 * 60
```

Comment:
This seems like a lot. I get the rationale about some things taking a long time to spin up - is there a way to differentiate between "not alive yet, working on it" and "errored out" though? If not, this is fine, if so, consider if it makes sense to implement that distinction to avoid a 15 minute delay between someone uploading erroneous code, and finding out it's erroneous.

Reply:
I know 15 minutes is a lot, but we observed real models that took over 8 minutes just to pull the image by Kubernetes. On the other hand, "cluster errors" happen quite rarely. If a job fails, it usually does on initialization step, which waits until the job is "ready" (with 10 minutes timeout).

> consider if it makes sense to implement that distinction

Good thinking, there's already an issue for that https://github.com/TheRacetrack/racetrack/issues/437